src/test/scala/nl.knaw.dans.easy.bagstore/server/StoresServletSpec.scala

```diff
@@ -353,10 +353,15 @@ class StoresServletSpec extends TestSupportFixture
     List("Authorization" -> s"$authType $encoded")
   }
 
+  def authenticationHeaderAndContentTypeZip(username: String, password: String): List[(String, String)] = {
+    authenticationHeader(username, password) ::: List(("Content-Type", "application/zip"))
+  }
+
   private val basicAuthentication = authenticationHeader(username, password)
+  private val basicAuthenticationAndZipContentType = authenticationHeaderAndContentTypeZip(username, password)
```

Comment:
just curious, why not inline that method call?

```scala
private val basicAuthenticationAndZipContentType = authenticationHeader(username, password) :+ "Content-Type" -> "application/zip"
```

Reply:
Simple, forgot to do that